internal/distro/rhel9/pipelines.go

```diff
@@ -933,6 +933,9 @@ func simplifiedInstallerTreePipeline(repos []rpmmd.RepoConfig, packages []rpmmd.
 		dracutStageOptions.Install = []string{"/fdo_diun_pub_key_root_certs.pem"}
 	}
 	p.AddStage(osbuild.NewDracutStage(dracutStageOptions))
+	if len(greenboot.MonitorServices) != 0 {
+		p.AddStage((osbuild.NewGreenbootConfig(new(osbuild.GreenbootConfig))))
```

Comment:
The `greenboot.MonitorServices` aren't added to the stage options.

Reply:
added them.